config/crds/migration_v1alpha1_directpvmigrationprogress.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: directpvmigrationprogresses.migration.openshift.io
+spec:
+  group: migration.openshift.io
+  names:
+    kind: DirectPVMigrationProgress
```

Comment:
Based on the name, this CR doesn't run the PV migration, it just reflects progress. Which controller is responsible for starting the Direct PV migration?

Reply:
I just noticed the other controller seems to be `directvolumemigration`, why the name convention change between these two? Seems like we should settle on using `PV` or `Volume` in both if they are related.

Reply:
> Based on the name, this CR doesn't run the PV migration, it just reflects progress. Which controller is responsible for starting the Direct PV migration?

@djwhatle https://github.com/konveyor/mig-controller/pull/732/files#diff-78a6bcf4a126e869712d026733ae2193015d729e56cbd9a3c49a14d8a6f6999cR117